.github/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# anything with no explicit code owners will be tagged to @core-devs
+*                       @napari/core-devs
+
+# submodules
+napari/_vispy/          @brisvag @melonora
+napari/_qt/             @Czaki @lucyleeow @DragaDoncila
+napari/_app_model/      @lucyleeow
+napari/benchmarks/      @Czaki
+napari/plugins/         @Czaki @DragaDoncila @lucyleeow
+napari/qt/              @Czaki
+napari/settings/        @Czaki @jni
```

Comment:
Oh man I don't like this 🤣 definitely reminding me of

<img width="489" alt="Screenshot 2024-07-24 at 3 35 35 PM" src="https://github.com/user-attachments/assets/75b913e8-c35a-4f48-b345-23487c2afb46">